populate-workflows.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -ex
+cd "${1:?Usage: $0 repo-dir}"
+mkdir -p .github/workflows
```

Comment:
```suggestion
test -e .git
mkdir -p .github/workflows
```

or some other `test` so that elderlies who manage to run it into some non-git repo which would have no point to have `.github` there.